src/Utilities/Hooks/useQuery.js

```diff
@@ -45,11 +45,9 @@ const useQuery = (fn, options = {}) => {
   const [meta, setMeta] = useState(undefined);
   const [error, setError] = useState(undefined);
   const [loading, setLoading] = useState(false);
-  console.log(params, 'params outside useQuery here');
-
+  const [availableIDs, setAvailableIDs] = useState([]);
```

Comment:
Remember what I said earlier today. The `useQuery` should only be concerned with calling a function and returning the results either as a state or directly.

It is not meant to alter, separate out or do anything with the result.
It shouldn't know about `meta` either. These are things that if anything the `useRemediationsQuery` should be responsible for.

Reply:
Ahh youre right, I was about to change it to also just bring back data, instead of data.data, and then use meta with data.meta, but im gonna go ahead with the compileResult thing instead